addons/comments-linebreaks/addon.json

```diff
@@ -7,6 +7,12 @@
       "link": "https://scratch.mit.edu/users/TheColaber/"
     }
   ],
+  "info": [
+    {
+      "text": "The line breaks are only visible to you. Line breaks in your own comments will not appear as new lines to everyone.",
```

Comment:
I think this could be worded better. It seems to make more sense with the first "the" left out in my opinion.

Reply:
I agree.

Also, the current wording incorrectly implies that others with the addon enabled won't see the long

Reply:
What about something like "Line breaks are only visible to users with this addon enabled. They will appear as spaces for others."